regen-network/regen-web
# fix: APP-126 adding multiple files at once to MediaForm gallery photos

## Description

https://regennetwork.atlassian.net/browse/APP-126?atlOrigin=eyJpIjoiMmRmM2Q0NzE0YTEwNDU2YmEwMWEwMmQxYWU4ZTczYzUiLCJwIjoiaiJ9



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



1. From https://deploy-preview-2357--regen-marketplace.netlify.app/, log in
2. Edit or create a project
3. From the media form, select/drop multiple images at once to the gallery photos and add caption and/or credit
4. Captions/credits should be filled in correctly for all images

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
